Remove Travertine (#63)
PaperMC/PaperDocs: source/site/api.rst

--------------------------------------
To download the latest jar, simply connect to `<https://papermc.io/api/v1/paper/1.16.3/latest/download>`__

> If you're looking for Waterfall, it's name can be used by replacing
> `paper` with `waterfall`. Specific versions can also be

Reviewer: `it's` -> `its`, but also the whole wording here is awkward.
Probably would be better as something like
```suggestion
If you're looking for Waterfall, the name `paper` can be
replaced with `waterfall`. Specific versions can also be
```
Also probably out of scope for this PR but I thought api v1 was deprecated.

Reviewer 2: We're probably gonna redo the entire page at some point. The pressure building each day we get closer to v1 being gone.

Author: I wanted to update references to api v1 but I felt like that was too much of a major change to make in one PR.

Reviewer 3: v1 is also currently used elsewhere in the documentation, would likely be better to remove it all at once rather than have some referencing v1 and some v2.
